src/client/vue/pages/player/PageSettings.vue

```diff
@@ -3,15 +3,21 @@ import { BIconBrightnessHighFill, BIconMoonStarsFill, BIconCircleHalf, BIconPcDi
 import useDarkLightThemeStore from '../../../stores/darkLightThemeStore';
 import { storeToRefs } from 'pinia';
 import usePlayerSettingsStore from '../../../stores/playerSettingsStore';
-import { watch } from 'vue';
+import useAuthStore from '../../../stores/authStore';
+import { ApiClientError } from '../../../apiClient';
+import { watch, Ref, ref } from 'vue';
 import { useSeoMeta } from '@unhead/vue';
+import { InputValidation, toInputClass } from '../../../vue/formUtils';
 
 useSeoMeta({
     robots: 'noindex',
     titleTemplate: title => `Settings - ${title}`,
 });
 
 const playerSettingsStore = usePlayerSettingsStore();
+const authStore = useAuthStore();
+
+const hasAccount = !(authStore.loggedInPlayer?.isGuest ?? true);
```

Comment:
This sometimes does not work because of a race condition: when you refresh the settings page, sometimes the "Change password" section not shows.

It is because at this moment, `loggedInPlayer` is not yet loaded.

The solution is to get a reactive value of `loggedInPlayer` with `storeToRefs()`:

```suggestion
const { loggedInPlayer } = storeToRefs(useAuthStore());
```

Then, in the `v-if` below, doing:

``` js
v-if="loggedInPlayer && !loggedInPlayer.isGuest"
```

This way, once the user is loaded and `loggedInPlayer` is updated, the reactivity is conserved up to here, vue will react to this change and update the view.

(I tried keeping the `hasAccount` function but it seems not reactive, not sure why).